.github/workflows/release.yml

```diff
@@ -29,25 +32,146 @@ jobs:
         run: |
           hatch build -t wheel
 
+      - name: set environment variables
+        id: set_env
+        run: |
+          OTACLIENT_WHL=$(basename $(ls dist/otaclient-*.whl))
+          OTACLIENT_VERSION=$(echo $OTACLIENT_WHL | sed -E 's/otaclient-([0-9]+\.[0-9]+(\.[0-9]+)?).*\.whl/\1/')
```

Comment:
You can use `hatch version` to get the built otaclient package version.

Reply:
Thank you, fixed.